configs/components/openssl-3.0.rb

```diff
@@ -121,9 +121,6 @@
     'no-whirlpool'
   ]
 
-  # Remove this in 3.0.15 or later
-  pkg.apply_patch 'resources/patches/openssl/CVE-2024-5535.patch'
```

Comment:
If this patch is no longer needed, could you `git rm` it?

Reply:
This patch also being used for openssl v1.1.1